### PR TITLE
Update code owners

### PR DESCRIPTION
**Pull request by srnagar**

### CODE REVIEW

**Review by ki1729 — APPROVED** [review]
🚢

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -133,10 +133,13 @@
 /sdk/loganalytics/microsoft-azure-loganalytics/     @divyajay @alongafni
 
 # PRLabel:  %Monitor - Exporter
-/sdk/monitor/azure-monitor-opentelemetry-exporter   @trask @kryalama @ramthi @heyams
```
Comment (jsquire) [inline .github/CODEOWNERS:136]: @srnagar :  Is this removal from one of our partner teams?   I'm not sure if Ronnie and I need to adjust "Service Attention" bot rules to reflect this change.

Comment (srnagar) [reply .github/CODEOWNERS:136]: @jsquire yes, this is from one of our partner teams. The bot rules have to be updated too. If it's something I can update, please let me know.

**Pull request merged**